c2cgeoform/.project

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
You should not commit eclipse project file and settings. I ignore them using https://help.github.com/articles/ignoring-files/#create-a-global-gitignore